bootstrap.sh

```diff
@@ -9,7 +9,11 @@ indent() {
 }
 
 indent_cli() {
-	sed -u 's/^/   > /'
+  if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Could you indent with tabs, so it matches the rest of the file? ;)

Reply:
Should be fixed.